recipes/pyemf/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyemf" %}
+{% set version = "2.1.2-alpha" %}
```

Comment:
This will need to target the pre-release channel. 

https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
The repository has been updated and the version is not alpha anymore